config/mbed/chip-gn/args.gni

```diff
@@ -21,6 +21,11 @@ chip_project_config_include = ""
 chip_system_project_config_include = ""
 chip_device_project_config_include = ""
 
+chip_custom_build_cflags = [
+    "-Wno-shadow",
+    "-Wno-conversion"
```

Comment:
What do you think about adding
-Wno-maybe-uninitialized
-Wno-return-type
also?

Reply:
I guess these new flags may be helpful when connectivityManagerImpl is used.

Reply:
Also, but mainly integration CHIP with mbed code reveals these warnings